applications/nrf_desktop/sysbuild/CMakeLists.txt

```diff
@@ -25,6 +25,12 @@ set(ipc_radio_APPLICATION_CONFIG_DIR
     CACHE INTERNAL "Application configuration dir controlled by sysbuild"
 )
 
+if (BOARD MATCHES "nrf52810dmouse/nrf52810" OR
```

Comment:
If possible, I would suggest to apply the DTS and Kconfig changes explicitly in nRF Desktop configurations (seems that only DTS needs to be slightly aligned). nRF Desktop maintains configurations per board and build type. With the proposed approach, allowing another board to use SW Split LL would require modifying common `CMakeLists.txt` (which we want to avoid if possible). Also it would not be possible for a board to use SW Split LL in one configuration and use SoftDevice LL in another configuration.

As an alternative, maybe we could consider introducing a sysbuild option to allow enabling the snippet through sysbuild? Thanks to that, we could explicitly enable it only in configurations that actually want to use SW Split LL.

Reply:
Discussed offline. Will manually apply overlays to all configuration files that need to use bt-ll-sw-split (for nrf_desktop only)